sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -557,7 +555,9 @@ def assert_counter_exists(metrics, namespace, name, step):
   # this test is flaky when state duration is low.
   # Since increasing state duration significantly would also slow down
   # the test suite, we are retrying twice on failure as a mitigation.
-  @retry(reraise=True, stop=stop_after_attempt(3))
+  @retry.with_exponential_backoff(num_retries=2, factor=1,
+                                  initial_delay_secs=0.1, max_delay_secs=0.1,
+                                  retry_filter=(lambda e: e is not None))
```

Comment:
Do we need a retry filter?

Reply:
Yes, The default retry filter checks for http errors while here we want to check for all errors.